app/services/description_generator.rb

```diff
@@ -0,0 +1,81 @@
+# typed: true
+# frozen_string_literal: true
+
+# This generates a RequestDRO Description for a work
+class DescriptionGenerator
+  extend T::Sig
+
+  sig { params(work: Work).returns(Hash) }
+  def self.generate(work:)
+    new(work: work).generate
+  end
+
+  sig { params(work: Work).void }
+  def initialize(work:)
+    @work = work
+  end
+
+  sig { returns(Hash) }
+  def generate
+    {
+      title: title,
+      # TODO: keywords not yet in model.
+      note: [abstract, citation],
+      event: [created_date],
+      access: {
```

Comment:
Are you sure about this?  In current hydrus objects, this is just in the notes as:
```
  <note type="contact" displayLabel="Contact">aydin@stanford.edu</note>
```

Reply:
What does "sure" mean? This was my best guess. Happy to change.

Reply:
I don't know where you are getting your idea to do access here. I thought maybe your information was better than mine.  My thought was to copy Hydrus unless Arcadia tells us otherwise.